scripts/zfs-tests.sh

```diff
@@ -455,7 +464,7 @@ if [ "$(id -u)" = "0" ]; then
 fi
 
 if [ "$(sudo whoami)" != "root" ]; then
-	fail "Passwordless sudo access required."
+	skip "Passwordless sudo access required."
```

Comment:
Doesn't this ask for a password, if sudo is not configured? I think you can use `sudo -n true` and return 0 is sudo-with-password is OK.